ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/MultiMap.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  ******************************************************************************
+ *    Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *    Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *    this file except in compliance with the License. A copy of the License is located at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file.
+ *    This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *    specific language governing permissions and limitations under the License.
+ *  ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.utils;
+
+import java.util.Collection;
+import java.util.Map;
+
+public interface MultiMap<K, V> {
```

Comment:
We can't add the functional interface annotation to this in java7, but we will be adding more methods to this as we need them.  This was the only method that I needed now.